data/mappings/info_config.hjson

```diff
@@ -125,6 +125,29 @@
     "MOUSEKEY_TIME_TO_MAX": {"info_key": "mousekey.time_to_max", "value_type": "int"},
     "MOUSEKEY_WHEEL_DELAY": {"info_key": "mousekey.wheel_delay", "value_type": "int"},
 
+    // OLED
+    "OLED_BRIGHTNESS": {"info_key": "oled.brightness", "value_type": "int"},
+    "OLED_COLUMN_OFFSET": {"info_key": "oled.column_offset", "value_type": "int"},
+    "OLED_DISPLAY_CLOCK": {"info_key": "oled.display_clock", "value_type": "hex"},
+    "OLED_FONT_H": {"info_key": "oled.font.path", "value_type": "str"},
```

Comment:
Really not a fan of introducing filenames into the json format. It sets a bad precedence for other configuration as well as poorly encapsulates the general problem being solved.

Primarily `OLED_FONT_H` is used to update the "default" font with logos. These would be better served by using something like <https://docs.qmk.fm/features/oled_driver#logo-example> and the reuse of the default font file.